StringIndexOutOfBoundsException

Please see https://github.com/originalrusyn/bug/tree/StringIndexOutOfBoundsException to reproduce this issue and  https://bugs.openjdk.java.net/browse/JDK-8214775. 
I’ve commit https://github.com/originalrusyn/bug/blob/StringIndexOutOfBoundsException/src/main/resources/javadoc.options generated file too. 
Please try to run "./gradlew clean assemble " multiple times to reproduce this issue. 

A DESCRIPTION OF THE PROBLEM : 
```
javadoc: error - fatal error encountered: java.lang.StringIndexOutOfBoundsException 
javadoc: error - Please file a bug against the javadoc tool via the Java bug reporting page 
(http://bugreport.java.com) after checking the Bug Database (http://bugs.java.com) 
for duplicates. Include error messages and the following diagnostic in your report. Thank you. 
java.lang.StringIndexOutOfBoundsException 
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) 
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490) 
at java.base/java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:603) 
at java.base/java.util.concurrent.ForkJoinTask.reportException(ForkJoinTask.java:678) 
at java.base/java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:737) 
at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateParallel(ForEachOps.java:159) 
at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel(ForEachOps.java:173) 
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233) 
at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497) 
at capital.scalable.restdocs.jsondoclet.ExtractDocumentationAsJsonDoclet.run(ExtractDocumentationAsJsonDoclet.java:58)
at jdk.javadoc/jdk.javadoc.internal.tool.Start.parseAndExecute(Start.java:582) 
at jdk.javadoc/jdk.javadoc.internal.tool.Start.begin(Start.java:431) 
at jdk.javadoc/jdk.javadoc.internal.tool.Start.begin(Start.java:344) 
at jdk.javadoc/jdk.javadoc.internal.tool.Main.execute(Main.java:63) 
at jdk.javadoc/jdk.javadoc.internal.tool.Main.main(Main.java:52) 
Caused by: java.lang.StringIndexOutOfBoundsException: begin 0, end 54, length 47 
at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3319) 
at java.base/java.lang.String.substring(String.java:1874) 
at jdk.compiler/com.sun.tools.javac.tree.DocTreeMaker.splitBody(DocTreeMaker.java:547) 
at jdk.compiler/com.sun.tools.javac.tree.DocTreeMaker.newDocCommentTree(DocTreeMaker.java:215) 
at jdk.compiler/com.sun.tools.javac.parser.DocCommentParser.parse(DocCommentParser.java:145) 
at jdk.compiler/com.sun.tools.javac.parser.LazyDocCommentTable.getCommentTree(LazyDocCommentTable.java:84)
at jdk.compiler/com.sun.tools.javac.api.JavacTrees.getDocCommentTree(JavacTrees.java:808) 
at jdk.compiler/com.sun.tools.javac.api.JavacTrees.getDocCommentTree(JavacTrees.java:820) 
at capital.scalable.restdocs.jsondoclet.FieldDocumentation.fromFieldDoc(FieldDocumentation.java:45) 
at capital.scalable.restdocs.jsondoclet.ClassDocumentation.addField(ClassDocumentation.java:63) 
at capital.scalable.restdocs.jsondoclet.ClassDocumentation.lambda$fromClassDoc$0(ClassDocumentation.java:47)
at java.base/java.lang.Iterable.forEach(Iterable.java:75) 
at capital.scalable.restdocs.jsondoclet.ClassDocumentation.fromClassDoc(ClassDocumentation.java:45) 
at capital.scalable.restdocs.jsondoclet.ExtractDocumentationAsJsonDoclet.lambda$run$1(ExtractDocumentationAsJsonDoclet.java:63)
at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) 
at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177) 
at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948) 
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) 
at java.base/java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:290) 
at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746) 
at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290) 
at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020) 
at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656) 
at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594) 
at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177) 
2 errors 
```

--- a/spring-auto-restdocs-json-doclet-jdk9/src/main/java/capital/scalable/restdocs/jsondoclet/ExtractDocumentationAsJsonDoclet.java
+++ b/spring-auto-restdocs-json-doclet-jdk9/src/main/java/capital/scalable/restdocs/jsondoclet/ExtractDocumentationAsJsonDoclet.java
@@ -53,7 +53,7 @@ public class ExtractDocumentationAsJsonDoclet extends StandardDoclet {
         ObjectMapper mapper = createObjectMapper();
 
         docEnv.getIncludedElements()
-                .stream().parallel()
+                .stream()
                 .filter(e -> e.getKind().isClass() || e.getKind().isInterface())
                 .forEach(classOrInterface -> writeToFile(
                         destinationDir,

